portal-web/test/functional/com/liferay/portalweb/macros/Publications.macro

```diff
@@ -12,7 +12,7 @@ definition {
 			key_publicationName = "${publicationName}",
 			locator1 = "Publications#PUBLISHED_PUBLICATION_NAME");
 
-		Pause(locator1 = "1000");
+		WaitForElementPresent(locator1 = "PublicationsChanges#CHANGE_HEADER");
```

Comment:
I agree with your idea, but for this assertion why I didn't assert the specific element for the next step due to it will direct to the changes page and I'd like to assert the changes page display firstly, do you think it makes sense?

Reply:
Makes sense to me! Thanks for your explanation.